internal/provider/resource_ger_ruleset_rule.go

```diff
@@ -0,0 +1,249 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/squadcast/terraform-provider-squadcast/internal/api"
+	"github.com/squadcast/terraform-provider-squadcast/internal/tf"
+)
+
+func resourceGERRulesetRule() *schema.Resource {
+	return &schema.Resource{
+		Description: "GER Ruleset Rule is basically a rule that is used to route alerts matching an expression to the correct service based on the alert source.",
+
+		CreateContext: resourceGERRulesetRuleCreate,
+		ReadContext:   resourceGERRulesetRuleRead,
+		UpdateContext: resourceGERRulesetRuleUpdate,
+		DeleteContext: resourceGERRulesetRuleDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceGERRulesetRuleImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Description: "GER Ruleset id.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"ger_id": {
+				Description: "GER id.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"description": {
+				Description: "GER Ruleset Rule description.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"alert_source": {
+				Description: "GER Ruleset alert source. Find all alert sources [here](https://www.squadcast.com/integrations).",
```

Comment:
The link provided is more of doc. I believe there was another API which was just showing the JSON data of all supported alert sources.